packages/webkit2gtk_4.rb

```diff
@@ -27,7 +21,7 @@ class Webkit2gtk_4 < Package
   depends_on 'enchant' # R
   depends_on 'fontconfig'
   depends_on 'freetype' # R
-  depends_on 'gcc10' => :build if %w[aarch64 armv7l i686].include? ARCH
+  depends_on 'gcc10' => :build if %w[aarch64 armv7l x86_64].include? ARCH
```

Comment:
```
depends_on 'gcc10' => :build unless ARCH == 'i686'
```
?

Reply:
Actually, doesn't `compatibility 'x86_64'` make this logic unnecessary?

Reply:
Just removed the condition.